andela/ah-frontend-thanos
# #161348763 Article tags should be present in all relevant views

#### What does this PR do?
Users can view the various article tags on an article.
A user can click on view all the tags available.

#### Description of Task to be completed?
- Users can view the various article tags on an article.
- A user can click on view all the tags available.


#### How should this be manually tested?
- Go to the article's URL: `/articles/:articleId` e.g. `/articles/40` and `/articles`
- You should be able to see the  article tags  on the article
- You should see the popular articles.

#### Any background context you want to provide?
You do need to be logged in to add a tag to an article
#### What are the relevant pivotal tracker stories?
[#161348763](https://www.pivotaltracker.com/story/show/161348763)

#### Relevant Screenshots
<img width="1458" alt="screen shot 2018-12-18 at 9 53 02 am" src="https://user-images.githubusercontent.com/22786444/50137114-37c79280-02ab-11e9-8ae9-c1b20d6c5f3b.png">
